src/examples/blinkm/blinkm.go

```diff
@@ -11,27 +11,26 @@ func main() {
 	machine.I2C0.Configure(machine.I2CConfig{})
 
 	// Init BlinkM
-	machine.I2C0.WriteTo(0x09, []byte("o"))
+	machine.I2C0.WriteRegister(0x09, '0', nil)
```

Comment:
This should be the letter "o" not the number 0, according to the BlinkM docs.

Reply:
Oops! fixed.